libraries/ESP8266HTTPClient/src/ESP8266HTTPClient.h

```diff
@@ -130,6 +130,12 @@ typedef enum {
     HTTPC_TE_CHUNKED
 } transferEncoding_t;
 
+typedef enum {
```

Comment:
This requires a Big Phat Comment explaining what each means (e. g. your pr explanation). At a glance, I can't tell the difference between follow and force. (I'm thinking of looking at this in the future).
Also, maybe better naming? Although I can't come up with anything better off the top of my head. If you can't either, just leave it as it is.

Reply:
See `af6f69a` commit

Reply:
@devyte any further work is required? PTAL

Reply:
Still not clear at a glance. Quoting the rfc is good as authoritative reference, but a minimal summary of the meanings should be here to not have to go look it up and dig through it.

Reply:
e. g. : strict means get redirects to get and put to put, force means <insert meaning here> despite not being rfc compliant because <insert reason here>.

Reply:
See https://github.com/esp8266/Arduino/pull/7157/commits/9510d166e74aaa567804a38ac96857ce05fce30f . I don't think that can be more explanatory, without citing RFC or something. Also, most users (as default) will use RFC strict mode. I think Google will also do the job, if someone is looking for ["how to follow post redirection for httpclient in esp8266"](http://google.com/search?q=how%20to%20follow%20post%20redirection%20for%20httpclient%20in%20esp8266).